src/components/ont/OntFlowcell.vue

```diff
@@ -69,4 +80,22 @@ export default {
   .empty {
     fill: red;
   }
+  .active {
+    stroke: white;
+    stroke-width: 2px;
+  }
+  foreignObject {
+    overflow: visible;
+  }
+  input{
```

Comment:
This input css seems a little overkill, is it all required?

Reply:
Yes, the transforms allow different browsers to rotate the input box respectively and the physical positioning keeps the input box within flowcell.